app/controllers/verify_controller.rb

```diff
@@ -43,7 +43,11 @@ def first_step
   end
 
   def enabled_steps
```

Comment:
looks like this method was previously unused? if not, might want to confirm no breaking changes.

Reply:
It's a private method previously used in a few other places in the controller.